static/app/icons/iconCellSignal.tsx

```diff
@@ -0,0 +1,26 @@
+import {forwardRef} from 'react';
+import {useTheme} from '@emotion/react';
+
+import {SvgIcon, type SVGIconProps} from 'sentry/icons/svgIcon';
+
+interface Props extends SVGIconProps {
+  bars?: 0 | 1 | 2 | 3;
+}
+const IconCellSignal = forwardRef<SVGSVGElement, Props>(({bars = 3, ...props}, ref) => {
```

Comment:
Not attached to this name, feel free to give suggestions.

Reply:
Looks like material icons [has a similar name](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:signal_cellular_alt:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=signal&icon.size=24&icon.color=%235f6368), so seems okay to me.

Only other think I could think of would be IconSignalBars